src/Analyzer.TemplateProcessor/ArmTemplateProcessor.cs

```diff
@@ -387,9 +387,13 @@ private void ProcessTemplateResourceLanguageExpressions(TemplateResource templat
                 if (templateResource.Properties != null)
                 {
                     evaluationHelper.OnGetCopyContext = () => templateResource.CopyContext;
+                    InsensitiveHashSet evaluationsToSkip = new InsensitiveHashSet();
+                    evaluationsToSkip.Add("template");  // The tool should skip properties in nested templates to avoid false positive warnings
```

Comment:
It's possible other resource types might have their own 'template' property for other uses.  This should only be excluded if the resource is a deployment type.